recipes/windows_reboot.rb

```diff
@@ -2,11 +2,7 @@
 node.default['windows']['allow_pending_reboots'] = true
 node.default['windows']['allow_reboot_on_failure'] = true
 
-include_recipe 'windows::reboot_handler'
-
-windows_reboot 'powershell' do
-  reason 'Reboot after successful/unsuccessful powershell installation'
-  timeout node['powershell']['reboot_timeout_seconds']
+reboot 'powershell' do
   action :nothing
   notifies :run, 'ruby_block[end_chef_run]', :immediately if node['powershell']['installation_reboot_mode'] == 'immediate_reboot'
```

Comment:
This notification to end the chef run isn't needed since we are passing the desired behavior to the resource via the `notifies` in the other recipes (passing `:reboot_now` tells it to end the Chef run cleanly and reboot right away, where `:request_reboot` would wait until the end of the chef run to reboot the node.  

I'd ditch the "reboot at the end of the run" behavior, as after the wmf install things are in a shaky state.